app-devices/switcheroo-control/autobuild/defines

```diff
@@ -0,0 +1,3 @@
+PKGNAME=switcheroo-control
+PKGSEC=utils
+PKGDES="D-Bus service to check the availability of dual-GPU"
```

Comment:
Missing dependencies, this is clearly an arch-specific package with dynamic dependencies.